### PR TITLE
[12.0][MIG] report_context

### PR DESCRIPTION
**Pull request by jarroyomorales**
Standard migration

### CODE REVIEW

File: report_context/views/ir_actions_report.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2019 Creu Blanca
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+    <record model="ir.ui.view" id="ir_actions_report_form_view">
```
Comment (tbaden) [inline report_context/views/ir_actions_report.xml]: optional:
id before model
and 
use original id (act_report_xml_view)

**Review by tbaden — APPROVED** [review]
thanks for the work 👍 
I did a code review: LGTM
If you like an optional change:

**Review by etobella — APPROVED** [review]
LGTM :+1: Review comments please :wink:

**Comment by jarroyomorales** [comment]
@etobella @tbaden Sorry, I didnt see the comment, just changed it.

**Pull request closed without merging**